tests/foreman/ui/test_repository.py

```diff
@@ -268,6 +268,74 @@ def test_positive_create_repo_with_checksum(self):
                         repo_name, 'checksum', checksum))
 
     @run_only_on('sat')
+    @tier1
+    def test_positive_create_repo_with_upstream_credentials(self):
+        """Create repository with upstream username and password parameters
+
+        :id: e6ca6e08-a2f7-4a67-9499-c4b12815d916
+
+        :expectedresults: Repository is created with expected username and
+            password field is hidden
+
+        :CaseImportance: Critical
+        """
+        repo_name = gen_string('alpha')
+        repo_username = gen_string('alpha')
+        product = entities.Product(organization=self.session_org).create()
+        with Session(self) as session:
+            set_context(session, org=self.session_org.name)
```

Comment:
Isn't it redundant? session org should be selected by default or it's not working for some reason?